<?xml version="1.0"?>
<!-- SESSION ID: 1510982379 -->
<launch>

  <!-- Checkerboard parameters -->
  <arg name="rows"          default="6" />
  <arg name="cols"          default="5" />
  <arg name="cell_width"    default="0.12" />
  <arg name="cell_height"   default="0.12" />

  <!-- Camera parameters -->
  <arg name="camera_name"     default="usb_0" />
  <arg name="camera_device"   default="1" />

  <arg name="intrinsic"   default="usb_0_intrinsic.yaml" />

  <!-- Launching camera -->
  <node pkg="ros_visual_localization" type="visual_localization" name="usb_0" output="screen">

    <param name="camera_device"       value="${arg camera_device}" />
    <param name="camera_type"           value="usb" />
    <param name="camera_name"           value="${arg camera_name}" />
    <param name="intrinsic"             value="${arg intrinsic}" />
    <param name="rows"                  value="$(arg rows)" />
    <param name="cols"                  value="$(arg cols)" />
    <param name="cell_width"            value="$(arg cell_width)" />
    <param name="cell_height"           value="$(arg cell_height)" />

  </node>

</launch>
